doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -117,16 +117,39 @@ Matter
 Matter fork
 +++++++++++
 
-The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.2.0.1`` tag.
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``v1.3.0.0`` tag.
 
 The following list summarizes the most important changes inherited from the upstream Matter:
 
+* Added:
+
+   * Support for the Scenes cluster.
+     This enables users to control the state for devices, rooms, or their whole home, by performing multiple actions on the devices that can be triggered with a single command.
+   * Command batching that allows controller to batch multiple commands into a single message.
+     This allows to minimize the delay between execution of the subsequent commands.
+   * Extended beaconing feature that allows an accessory device to advertise Matter service over Bluetooth LE for a period longer than maximum time of 15 minutes.
+     This can be enabled using the :kconfig:option:`CONFIG_CHIP_BLE_EXT_ADVERTISING`.
+   * The Kconfig options :kconfig:option:`CONFIG_CHIP_ICD_LIT_SUPPORT`, :kconfig:option:`CONFIG_CHIP_ICD_CHECK_IN_SUPPORT`, and :kconfig:option:`CONFIG_CHIP_ICD_UAT_SUPPORT` to manage ICD configuration.
+   * New device types:
+
+     * Device energy management
```

Comment:
This isn't in the tables.

Reply:
Yes, this is because we have sort of two groups of device types. One are the devices that actually reflect the real products functionalities (like dishwasher, light bulb, etc) and the other are Utility device types, which describe more internal Matter related functionalities (for example OTA Requestor, Power Source, Bridged Node or Device Energy Management). So to sum up Device Energy Management is a device type, but not a "real" device, so I didn't put it on the list.